releases/1.1.1-SNAPSHOT.yml

```diff
@@ -0,0 +1,3 @@
+version: 1.1.1-SNAPSHOT
+rpm_version: 1.1.1~SNAPSHOT
+deb_version: 1.1.1~SNAPSHOT
```

Comment:
I think support for tilde was added in RHEL 6.4. https://bugzilla.redhat.com/show_bug.cgi?id=825087